srcpkgs/dolphin/template

```diff
@@ -1,7 +1,7 @@
 # Template file for 'dolphin'
 pkgname=dolphin
```

Comment:
Am I wrong or shouldn't this revbump target `dolphin-emu` and not `dolphin`?

Reply:
Indeed, thanks. I corrected that.